spec/dealership_spec.rb

```diff
@@ -0,0 +1,130 @@
+  end
+
+  describe '#total_value' do
+    it 'returns the total price of the cars in the dealership' do
+      dealership = Dealership.new("Acme Auto", "123 Main Street")
+      car_1 = Car.new("Ford Mustang", 1500, 36)
+      car_2 = Car.new("Toyota Prius", 1000, 48)
+      car_3 = Car.new("Toyota Tercel", 500, 48)
+      car_4 = Car.new("Chevrolet Bronco", 1250, 24)
+      dealership.add_car(car_1)
+      dealership.add_car(car_2)
+      dealership.add_car(car_3)
+      dealership.add_car(car_4)
+      expect(dealership.total_value).to eq(156000)
+    end
+  end
+
+  describe '#details' do
+    it 'returns the basic information of the dealership' do
+      dealership = Dealership.new("Acme Auto", "123 Main Street")
+      car_1 = Car.new("Ford Mustang", 1500, 36)
+      car_2 = Car.new("Toyota Prius", 1000, 48)
+      car_3 = Car.new("Toyota Tercel", 500, 48)
+      car_4 = Car.new("Chevrolet Bronco", 1250, 24)
+      dealership.add_car(car_1)
+      dealership.add_car(car_2)
+      dealership.add_car(car_3)
+      dealership.add_car(car_4)
+      expect(dealership.details).to eq({"total_value" => 156000, "address" => "123 Main Street"})
+    end
+  end
+
+  describe '#average_price_of_car' do
+    it 'returns the average price of all cars in inventory' do
+      dealership = Dealership.new("Acme Auto", "123 Main Street")
+      car_1 = Car.new("Ford Mustang", 1500, 36)
+      car_2 = Car.new("Toyota Prius", 1000, 48)
+      car_3 = Car.new("Toyota Tercel", 500, 48)
+      car_4 = Car.new("Chevrolet Bronco", 1250, 24)
+      dealership.add_car(car_1)
+      dealership.add_car(car_2)
+      dealership.add_car(car_3)
+      dealership.add_car(car_4)
+      expect(dealership.average_price_of_car).to eq("39,000")
+    end
+  end
+
+  describe '#cars_sorted_by_price' do
+    xit 'returns a list of cars from least to most expensive' do
+      dealership = Dealership.new("Acme Auto", "123 Main Street")
+      car_1 = Car.new("Ford Mustang", 1500, 36)
+      car_2 = Car.new("Toyota Prius", 1000, 48)
+      car_3 = Car.new("Toyota Tercel", 500, 48)
+      car_4 = Car.new("Chevrolet Bronco", 1250, 24)
+      dealership.add_car(car_1)
+      dealership.add_car(car_2)
+      dealership.add_car(car_3)
+      dealership.add_car(car_4)
+      expect(dealership.cars_sorted_by_price).to eq([car_3, car_4, car_2, car_1])
+    end
+  end
+
+  describe '#inventory_hash' do
+    xit 'returns/creates a hash of all inventory listed by make' do
+      dealership = Dealership.new("Acme Auto", "123 Main Street")
+      car_1 = Car.new("Ford Mustang", 1500, 36)
+      car_2 = Car.new("Toyota Prius", 1000, 48)
+      car_3 = Car.new("Toyota Tercel", 500, 48)
+      car_4 = Car.new("Chevrolet Bronco", 1250, 24)
+      dealership.add_car(car_1)
+      dealership.add_car(car_2)
+      dealership.add_car(car_3)
+      dealership.add_car(car_4)
+      expect(dealership.inventory_hash).to eq({"Ford"=> [car_1],"Toyota"=> [car_2, car_3],"Chevrolet"=> [car_4]})
+    end
+  end
```

Comment:
I appreciate you sticking to TDD to write up these tests before getting a chance to implement the code!